valio-master/Valio/ScheduleTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ScheduleTableViewController.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/5/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+
+class ScheduleTableViewController: UITableViewController {
+
+	lazy var 📅: NSArray = {
+		let path = NSBundle.mainBundle().pathForResource("valio", ofType: "json")!
+		let data = NSData(contentsOfFile: path, options: nil, error: nil)!
+		return NSJSONSerialization.JSONObjectWithData(data, options: nil, error: nil) as! NSArray
+	}()
+	
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+		navigationItem.titleView = UIImageView(image: UIImage(named: "Valio"))
+		
+		tableView.registerClass(ItemTableViewCell.self, forCellReuseIdentifier: "cell")
+		tableView.separatorStyle = .None
+    }
+	
+	override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+		return 📅.count
+	}
+
+    override func tableView(tableView: UITableView?, numberOfRowsInSection section: Int) -> Int {
+		let day = 📅[section] as! NSDictionary
+		let items = day["items"] as! NSArray
```

Comment:
I would use swift native arrays and dictionaries, then I wouldn't have to cast the values here - I would be sure of the type.

If it had to be `NS*` I would at least use `as?` and handle the error gracefully, e.g. returning 0 here